sessions/react-props/demo-end/src/App.js

```diff
@@ -8,9 +8,34 @@ export default function App() {
 
   return (
     <div>
-      <Pet sound="Meow" emoji="🐈" onPet={handlePet} isHungry />
-      <Pet sound="Woof" emoji="🐕" onPet={handlePet} />
-      <Pet sound="Squeak" emoji="🐁" onPet={handlePet} />
+      <Pet
+        sound="Meow"
+        emoji={
+          <span role="img" aria-label="A cat">
+            🐈
+          </span>
+        }
+        onPet={handlePet}
+        isHungry
+      />
+      <Pet
+        sound="Woof"
+        emoji={
+          <span role="img" aria-label="A dog">
+            🐕
+          </span>
+        }
+        onPet={handlePet}
+      />
+      <Pet
+        sound="Squeak"
+        emoji={
+          <span role="img" aria-label="A mouse">
+            🐁
+          </span>
+        }
+        onPet={handlePet}
+      />
```

Comment:
Instead of repeating the 'span` couple of times I' rather keep it in the `Pet` component. This way it's not so confusing and easier to handle from the `demo-start` version.

```suggestion
      <Pet sound="Meow" emoji="🐈" name="cat" onPet={handlePet} isHungry />
      <Pet sound="Woof" emoji="🐕" name="dog" onPet={handlePet} />
      <Pet sound="Squeak" emoji="🐁" name="mouse" onPet={handlePet} />
```

The `Pet` component

```
function Pet({ sound, emoji, name, onPet, isHungry }) {
  return (
    <div onClick={onPet}>
      {isHungry ? "Feed me!" : sound}{" "}
      <span role="img" aria-label={name}>
        {emoji}
      </span>
    </div>
  );
}
```